Microsoft.Toolkit.Uwp.SampleApp/SamplePages/InfiniteCanvas/InfiniteCanvasPage.xaml.cs

```diff
@@ -81,6 +83,60 @@ private void Load()
                     }
                 }
             });
+
+            SampleController.Current.RegisterNewCommand("Export max view as image", async (sender, args) =>
+            {
+                if (_infiniteCanvas != null)
+                {
+                    var savePicker = new Windows.Storage.Pickers.FileSavePicker
+                    {
+                        SuggestedStartLocation = Windows.Storage.Pickers.PickerLocationId.Desktop
+                    };
+                    savePicker.FileTypeChoices.Add("image/png", new List<string> { ".png" });
+                    savePicker.SuggestedFileName = "Infinite Canvas Max View";
+
+                    var file = await savePicker.PickSaveFileAsync();
+
+                    if (file != null)
+                    {
+                        CachedFileManager.DeferUpdates(file);
+                        using (var stream = new InMemoryRandomAccessStream())
+                        using (var fileStream = await file.OpenAsync(FileAccessMode.ReadWrite))
+                        {
+                            var offScreen = _infiniteCanvas.ExportMaxOffScreenDrawings();
+                            await offScreen.SaveAsync(stream, CanvasBitmapFileFormat.Png);
```

Comment:
See pull #2702, it's adding a BitmapFormat enum we should align with that.  Hopefully that PR should be checked-in soon.